### ISSUE TITLE
Kerberos Authentication support in MSO4SC

### ISSUE THREAD

**Issue by ncde** [MSO4SC/cloudify-hpc-plugin#22]
Hello, 
PDC Center for High Performance Computing at KTH has installed  singularity on the cluster tegner (https://www.pdc.kth.se/about/pdc-news/pdc-now-supports-singularity-on-tegner-1.795667) and we would like to access this resource (https://www.pdc.kth.se/hpc-services/computing-systems/tegner-1.737437) from the MSO4SC portal.

The main problem that we have right now is in order to access Tegner, Kerberos Authentication has to be used. This means the client that needs to access the server needs to have 

the package installed (https://web.mit.edu/kerberos/) 
obtain tickets using kinit -f username@PDC.KTH.SE or username@NADA.KTH.SE
then do: 
ssh -vv -o GSSAPIDelegateCredentials=yes -o GSSAPIKeyExchange=yes -o GSSAPIAuthentication=yes username@tegner.pdc.kth.se

In short the user should have a way to forward the kerberos tickets that they have on their computer to the MSO4SC portal and be able to access tegner resources.
Best,
